src/liballoc/collections/btree/node.rs

```diff
@@ -179,7 +179,7 @@ impl<K, V> Root<K, V> {
         NodeRef {
             height: self.height,
             node: self.node.as_ptr(),
-            root: ptr::null_mut(), // FIXME: Is there anything better to do here?
+            root: ptr::null(),
```

Comment:
I doubt that comment was about the unnecessary _mut for a marker::Immut NodeRef, but I think the question has timed out.